.github/workflows/packages-filebeat.yml

```diff
@@ -1,11 +1,17 @@
-name: Packages - Build filebeat package
+name: Packages - Build Filebeat Package
 on:
   workflow_dispatch:
     inputs:
       revision:
         description: 'Revision used to naming Filebeat package wazuh-filebeat-X.X.tar.gz'
         required: true
-
+  workflow_call:
+    inputs:
+      revision:
+        required: true
+        default: "0.0"
```

Comment:
If there is a default value why it is required?

Reply:
Note: All development packages should use revision 0 (in this case it should be 0.0

Reply:
This workflow only manages an input value. 

Should another parameter be added to manage the `development/stage` package name or is it still intended to be handled just by the revision field? In this case, I think the set `default` value is correct.

Reply:
Added b3121120804317e81bbb3eddd49656feb9519540 to fix `required` field.

Reply:
Ah yes, sorry, the default will set a value in the input field, and if empty, the required will show an error, it is ok

Reply:
If you set the input as required, the input will ask for a value if the field is empty, if you set the default value, the field will have a value in the UI field, but if you remove it, the input will show an error

If you set required as false and set a default value, then the input field will show the default value, and if you remove it and do not specify a value, then the input will set the default value internally, that is what I am referring, the default value has two functions:

1. Set the input value in the UI
2. Set the input value if the UI field is not set

That is why I am saying that it makes no sense to have a default value with a required input in this case